src/bf2path/README.md

```diff
@@ -1,9 +1,9 @@
 # bf2path
 
-Brainfuck is.. *okay if I need to explain to you what brainfuck is you shouldn't be here...*<br/>
-You should be [here](https://www.youtube.com/watch?v=dxJpGVaCAyU)
+Brainfuck is a well known esoteric programming language.<br/>
```

Comment:
Much better :)

Reply:
to you...
###### *i liked the old one better*

Reply:
;P